contracts/drafts/TokenVesting.sol

```diff
@@ -16,8 +16,8 @@ contract TokenVesting is Ownable {
   using SafeMath for uint256;
   using SafeERC20 for IERC20;
 
-  event Released(uint256 amount);
-  event Revoked();
+  event TokenReleased(address token, uint256 amount);
+  event TokenRevoked(address token);
```

Comment:
This is a nice catch.
What would you think of calling them:
Token*s*Released and TokenVestingRevoked?

Reply:
Hm, agreed, it's multiple tokens that are released/revoked, not the contract. Long live our naming chief!